app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::ContactsController < Api::V1::BaseController
+  def index
+    render json: Contact.order(created_at: :desc)
+  end
+
+  def enum
+    @contact_enum = {'corners': create_enum_pairs(ContactCorners),
+                     'departments': create_enum_pairs(ContactDepartments),
+                     'grades': create_enum_pairs(ContactGrades),
+    }
```

Comment:
__[RuboCop]__

Layout/MultilineHashBraceLayout: Closing hash brace must be on the same line as the last hash element when opening brace is on the same line as the first hash element.


<!-- Comment by [SideCI](https://sideci.com) -->